core/examples/luigi-sample-angular/e2e/tests/context-switcher.spec.js

```diff
@@ -96,6 +96,8 @@ describe('Context switcher', () => {
       'Remove Project'
     );
 
+    cy.wait(3000);
```

Comment:
why do we need it here?

Reply:
fixed failing e2e test, but it seems it is not necessary anymore.